docs/datagen/intro.md

```diff
@@ -0,0 +1,53 @@
+Data Generators
+===============
+
+Data generators are a way to programmatically generate the assets and data of your mods. It allows the definition of the contents of these files in your code and the automatic generation of these files, without worrying about the specification of the files.
+
+The data generator system is loaded by the main class `net.minecraft.data.Main`. Different arguments can be passed to it, to customize which mods' data are gathered, what existing files are considered, etc. The class responsible for data generation is `net.minecraft.data.DataGenerator`.
```

Comment:
It would be useful to call out the `runData` gradle task as the way to trigger a data generator run. This makes it sound like you should be calling the main method yourself

Reply:
Done, thank you for reminding me to do that!